src/site/content/en/blog/image-support-for-async-clipboard/index.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Image support for the async clipboard API
+subhead: Starting in Chrome 76, the Asynchronous Clipboard API now handles some images, in addition to text.
+authors:
+  - thomassteiner
+description: Chrome 76 adds expands the functionality of the Async Clipboard API to add support for png images. Copying and pasting images to the clipboard has never been easier.
+date: 2019-07-03
+updated: 2019-10-08
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - capabilities
+  - chrome76
+  - cutandcopy
+  - execcommand
+  - input
+  - clipboard
+hero: hero.jpg
+alt: Clipboard with shopping list
+---
+
+Chrome 66, added support for the
+[text portion](https://developers.google.com/web/updates/2018/03/clipboardapi)
+of the [Asynchronous Clipboard API](https://w3c.github.io/clipboard-apis/#async-clipboard-api).
+Chrome 76 has added support for images makes it easy to programmatically
```

Comment:
```suggestion
Chrome 76's added support for images makes it easy to programmatically
```

Reply:
@kaycebasques - can you give me context on this? To me it doesn't sound quite grammatically correct.

Reply:
I think Kayce's edit makes sense:

"Chrome 76's added support for images makes it easy to programmatically copy and paste PNG images."

Reply:
The original sentence had trouble grammar-ing (problem in bold) so I just edited it to make it grammar-ful

> Chrome 76 **has** added support for images **makes** it easy to programmatically...

Reply:
Option 3: Chrome 76 supports images for easily copying and pasting PNG images.

Fewer words too.

Reply:
Went with option 4.